server/service/src/programs/encounter/encounter_fields.rs

```diff
@@ -45,7 +45,7 @@ pub(crate) fn encounter_fields(
     let encounters = repository.query(pagination, Some(filter.clone()), sort)?;
     let doc_names = encounters
         .iter()
-        .map(|row| row.document_name.clone())
+        .map(|row| row.0.document_name.clone())
```

Comment:
I guess similar to the comment in graphql layer, I think Encounter is nicer with either struct or getters for encounter/program

Reply:
Yes agree, will change when touch the next time!